dockerfiles/debian-11/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM debian:9
+FROM debian:bullseye
```

Comment:
Isn't Debian 10 supported?

Reply:
Maybe not until it's finally released: https://hub.docker.com/_/debian

Reply:
Ah, I did mean 10, not 11, since I mistook the diff as if you had moved the lower bound from Debian 9 to 11 instead of 10 – I forgot that there already is a debian-10 directory.